roles/openshift_register_nodes/vars/main.yml

```diff
@@ -1,7 +1,6 @@
 ---
-openshift_cert_parent_dir: /var/lib/openshift
-openshift_cert_relative_dir: openshift.local.certificates
-openshift_cert_dir: "{{ openshift_cert_parent_dir }}/{{ openshift_cert_relative_dir }}"
-openshift_master_ca_dir: "{{ openshift_cert_dir }}/ca"
-openshift_master_ca_cert: "{{ openshift_master_ca_dir }}/cert.crt"
-openshift_master_ca_key: "{{ openshift_master_ca_dir }}/key.key"
+openshift_node_config_dir: /etc/openshift/node
+openshift_master_config_dir: /etc/openshift/master
```

Comment:
openshift_register_nodes is actually run on the master host (generated oo_first_master group).  If needed we should probably either duplicate these vars in the openshift_register_nodes role or move it to the local_facts on the master.